src/configs/mise.rs

```diff
@@ -0,0 +1,41 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(
+    feature = "config-schema",
+    derive(schemars::JsonSchema),
+    schemars(deny_unknown_fields)
+)]
+#[serde(default)]
+pub struct MiseConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+    pub detect_extensions: Vec<&'a str>,
+    pub detect_files: Vec<&'a str>,
+    pub detect_folders: Vec<&'a str>,
+    pub healthy_symbol: &'a str,
+    pub unhealthy_symbol: &'a str,
+}
+
+impl<'a> Default for MiseConfig<'a> {
+    fn default() -> Self {
+        Self {
+            format: "[$symbol$health]($style) ",
+            symbol: "mise ",
+            style: "bold purple",
+            disabled: true,
+            detect_extensions: vec![],
+            detect_files: vec![
+                "mise.toml",
+                "mise.local.toml",
+                ".mise.toml",
+                ".mise.local.toml",
+            ],
+            detect_folders: vec![".mise"],
+            healthy_symbol: "healthy",
+            unhealthy_symbol: "unhealthy",
```

Comment:
I feel like emoji could allow saving space here, but this is fine if you want to keep it like this.

Reply:
I will leave it as is unless another contributor wants to suggest emojis for these symbols.